.setup/vagrant/sites-available/submitty.conf

```diff
@@ -46,6 +46,17 @@
         Require all granted
         Order allow,deny
         Allow from all
+
+        RewriteEngine On
```

Comment:
If we were going to implement a capacity of turning on/off "pretty links" (e.g. /s19/sample vs ?semester=s19&course=sample), then we'd want to wrap this in a "<IfModule>" check, but I think we're going to make this a mandatory component of Submitty, so no need, and it'd be better for apache to just fail if the module wasn't enabled.